.github/workflows/cd-pdqm320.yml

```
@@ -0,0 +1,31 @@
+        - DEV
+        - PROD
+        required: true
+      ballerina_version:
+        description: Ballerina Version
+        type: choice
+        options:
+        - 2201.9.0
+        - 2201.10.4
+        - 2201.12.3
+        required: false
+        default: 2201.10.2
```

actionlint (1.7.12)

[error] default value "2201.10.2" of "ballerina_version" input is not included in its options "\"2201.9.0\", \"2201.10.4\", \"2201.12.3\""

(events)